llvm/test/MC/RISCV/option-exact-inlineasm.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -mtriple=riscv32 -mattr=+relax,+c %s --filetype=obj -o - \
```

Comment:
Move to test/CodeGen/RISCV?

Reply:
Done, yeah, this location was a little iffy, happier to have an `llc` test in the CodeGen directory, though this isn't the only one in the MC/RISCV directory